storage/fuzz/fuzz_targets/journal_operations.rs

```diff
@@ -1,12 +1,8 @@
 #![no_main]
 
 use arbitrary::{Arbitrary, Result, Unstructured};
-use commonware_cryptography::{Hasher as _, Sha256};
 use commonware_runtime::{buffer::PoolRef, deterministic, Runner};
-use commonware_storage::journal::fixed::{
-    Config as FixedConfig, Config as VariableConfig, Journal as FixedJournal,
-    Journal as VariableJournal,
```

Comment:
We were importing `fixed::Journal` under two different names and using it twice 😅 

Updated this fuzz test to operate over a `Journal`.